internal/host/validator.go

```diff
@@ -722,7 +722,8 @@ func (v *validator) validateNetworkLatencyForRole(host *models.Host, clusterRole
 				if _, ok := failedHostIPs[l3.RemoteIPAddress]; !ok {
 					hostname, role, err := GetHostnameAndRoleByIP(l3.RemoteIPAddress, hosts)
 					if err != nil {
-						return ValidationFailure, nil, fmt.Errorf("unable to determine role and hostname for IP %s: %s", l3.RemoteIPAddress, err)
+						v.log.Error(err)
```

Comment:
This log message will be meaningless without the context

Reply:
I figured that the incoming error message already contains sufficient information, wrapping it would not add any value:

https://github.com/openshift/assisted-service/blob/581f79c8b3b42dfe92467bfeaf644fde404fd2e4/internal/host/common.go#L103-L106

https://github.com/openshift/assisted-service/blob/581f79c8b3b42dfe92467bfeaf644fde404fd2e4/internal/host/common.go#L119-L120